packages/cli/__tests__/daemon/__integration__/cli.zowe.exe.integration.ts

```diff
@@ -34,27 +35,35 @@ describe("Zowe native executable", () => {
             skipProperties: true
         });
 
-        // determine executable file extension for our current OS
-        let exeExt: string;
+        // determine executable file name for our current OS
+        const zoweRootDir: string = nodeJsPath.normalize(__dirname + "/../../../../..");
         const sysInfo: ISystemInfo = ProcessUtils.getBasicSystemInfo();
         if (sysInfo.platform == "win32") {
-            exeExt = ".exe";
+            zoweExeFileNm = "zowe.exe";
+        } else if (sysInfo.platform == "linux") {
```

Comment:
Could this else if be dropped and just be the else?

Reply:
I think in both this case and your later comment, the code set another variable which was different for the different cases. After a refactoring which removed the second variable, I failed to notice the redundancy. These are good changes that I will implement.